Probe/src/probe/logger.go

```diff
@@ -81,7 +81,7 @@ func (c *CloudLogger) LogFatalf(desc string, args ...interface{}) {
 // Log errors to specified log
 func (c *CloudLogger) LogError(desc string) {
 	//TODO(langenbahn) add any other useful information for errors
-	el := &errorLog{desc, probeConfigs.Probe[0].GetRegion(), deviceToken}
+	el := &errorLog{desc, hostname, deviceToken}
```

Comment:
Where does hostname and deviceToken come from - should they be members of CloudLogger?

Reply:
addressed in ef6a24c